src/fides/api/ops/models/storage.py

```diff
@@ -103,3 +90,87 @@ def set_secrets(
 
         self.secrets = storage_secrets
         self.save(db=db)
+
+
+def default_storage_config_name(storage_type: str) -> str:
+    """
+    Utility method for consistency in generating default storage config names.
+
+    Returns a name to be used in a default storage config for the given type.
+    """
+    return f"Default Storage Config [{storage_type}]"
+
+
+def default_storage_config_by_type(
+    db: Session, storage_type: StorageType
+) -> Optional[StorageConfig]:
+    """
+    Retrieve the default storage config for the given type by querying for the
+    `StorageConfig` record with the given type and with `is_default` set to `True`.
+
+    There should only ever be one `StorageConfig` record that
+    matches this criteria at any point.
+
+    If `local` storage type is requested and a default doesn't yet exist, we create a default.
+    `local` storage configurations, at this point, don't have any configurable settings,
+    so we can safely create a default here, at the time it's requested.
+    """
+    if not isinstance(storage_type, StorageType):
+        # try to coerce into an enum
+        try:
+            storage_type = StorageType[storage_type]
+        except KeyError:
+            raise ValueError(
+                "storage_type argument must be a valid StorageType enum member."
+            )
```

Comment:
yeah, good call out. i was trying to make it easier for the caller, in the test cases and in `get_active_default_storage_config` we're potentially passing in a `str`. not 100% sure why mypy isn't complaining, i'm sure there's some good reason :)

we could do this coercion in the calling code, do you think that's better? alternatively i could make the typehint for `storage_type` a `Union[StorageType, str]`? from my POV it's nice to have the function handle this coercion so you don't have to think about it as much from the calling code, but it may be bad convention and make for less maintainable code, in which case i'm happy to go ahead and updating the calling code.